### PR TITLE
initial work on health monitoring

### PR DESCRIPTION
**Pull request by samdods**

### CODE REVIEW

File: OpenTrace/Main/View Controllers/HomeViewController.swift [hunk 1]
```diff
@@ -68,7 +69,7 @@ class HomeViewController: UIViewController {
 
     private func readPermissionsAndUpdateViews() {
 
-        blePoweredOn = BluetraceManager.shared.isBluetoothOn()
+        blePoweredOn = true//BluetraceManager.shared.isBluetoothOn()
```
Comment (Neil3079) [inline OpenTrace/Main/View Controllers/HomeViewController.swift]: Commited by mitake?

File: OpenTrace/Monitoring/Symptoms/SymptomsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//  Created by Sam Dods on 15/04/2020.
+//  Copyright © 2020 OpenTrace. All rights reserved.
+//
+
+import UIKit
+
+class SymptomsViewController: UIViewController {
+
+    convenience init() {
+        self.init(nibName: String(describing: SymptomsViewController.self), bundle: Bundle(for: SymptomsViewController.self))
```
Comment (Neil3079) [inline OpenTrace/Monitoring/Symptoms/SymptomsViewController.swift:14]: I like this even from the Main Bundle!!!!!!

**Review by Neil3079 — APPROVED** [review]

File: OpenTrace/Main/View Controllers/HomeViewController.swift (hunk 1 above)
Comment (samdods) [reply OpenTrace/Main/View Controllers/HomeViewController.swift]: correct

File: OpenTrace/Monitoring/Symptoms/SymptomsViewController.swift (hunk 2 above)
Comment (samdods) [reply OpenTrace/Monitoring/Symptoms/SymptomsViewController.swift:14]: haha, autopilot :)

**Pull request merged**